lib/widgets/editor/editor_toolbar.dart

```diff
@@ -0,0 +1,462 @@
+import 'package:flutter/foundation.dart';
```

Comment:
(unrelated to this import) Personally, I would move helpers to the bottom of the file. Since EditorToolbar is the main focus